src/Cms/PageRules.php

```diff
@@ -293,6 +297,25 @@ public static function update(Page $page, array $content = []): bool
             ]);
         }
 
+        self::slugMaxlength($page);
+
```

Comment:
If I understand the logic correctly, updating the content won't change the slug, so this method wouldn't need to validate the slug length. Or have I missed something?

Reply:
You right! I thought slug could be updated with the update() method, but this is not possible yes.